integration/package-operator/objectset_test.go

```diff
@@ -224,7 +223,7 @@ func runObjectSetSetupPauseTeardownTest(t *testing.T, namespace, class string) {
 		Waiter.WaitForObject(ctx, currentCM4, "to NOT be reconciled to its desired state", func(obj client.Object) (done bool, err error) {
 			cm := obj.(*corev1.ConfigMap)
 			return cm.Data["banana"] == "bread", nil
-		}, dev.WithTimeout(5*time.Second)), wait.ErrWaitTimeout.Error())
+		}, dev.WithTimeout(5*time.Second)), "timed out waiting for the condition")
```

Comment:
Replacement for ErrWaitTimeout deprecation: https://pkg.go.dev/k8s.io/apimachinery@v0.27.4/pkg/util/wait#Interrupted

Reply:
Oh, forgot about that. Thank you!